tests/functional/utils/model_management.py

```diff
@@ -40,6 +40,23 @@ def wait_endpoint_setup(container):
     time.sleep(2)
     return running
 
+def wait_minio_endpoint_setup(container):
```

Comment:
This function and `wait_endpoint_setup` from this file could be merged to one with a parameter as a message we want to have in logs.

Reply:
wait_endpoint_setup checks for specific line in logs and has a big 900 timeout while wait_minio_endpoint_setup checks container status and has smaller timeout thus I would prefer to leave it as separate functions.

Reply:
So the timeout can be parametrized too.
As long as `if` is checking for boolean value, still the conditions as well as timeout can be passed as parameters, so the signature will looks like`def wait_endpoint_setup(condition, timeout)`.

Reply:
Done.